project_timeline/views/project_task_view.xml

```diff
@@ -60,7 +60,7 @@
         <field name="arch" type="xml">
             <timeline
                 date_start="date_assign"
```

Comment:
You have to change this date, populating first in the SQL of the report the needed field